### DISCUSSION TITLE
[WASM] RichEditBox implementation

### DISCUSSION THREAD

**Discussion by SanAlexis** [unoplatform/uno#11611] [Q&A]
I'm doing a migration project from a project targeting UWP `uap10.0.17134.0` to a project targeting WASM `net6.0`. I'm unable to migrate a class implementing `RichEditBox` since it's not yet implemented in `Uno v4.6.19`. Is there a work around to realise this migration ?

**Comment by jeromelaban** [comment]
See https://github.com/unoplatform/uno/discussions/11244 for more details.
